core/src/main/resources/jenkins/model/Jenkins/help-rawBuildsDir.html

```diff
@@ -1,6 +1,11 @@
 <div>
-    Specifies where Jenkins will store build records on the file system. This
+    <p><strong>This value is only modifiable using a system property.
+        To do so, pass <code>-Djenkins.model.Jenkins.BUILDS_DIR="the_value"</code>
+        to the Jenkins JVM options during startup.</strong>
```

Comment:
Needs explanation. E.g. that Jenkins will not migrate existing records, and Jenkins will start up with empty ones.

Reply:
@daniel-beck this is not changing behaviour, though 100% agreed to write tests. Going to.

About "will not migrate existing records", it's already explained below, see https://github.com/jenkinsci/jenkins/pull/3360/files#diff-40eecaef8bbb59ebafb2cf96bfdb031cR29 : 

> note that Jenkins will not automatically migrate any data
     from the current build record root directory.	     from the current build record root directory